content/docs/concepts/traffic-management/overview/index.md

```diff
@@ -0,0 +1,154 @@
+   routing rules for HTTP, gRPC, WebSocket, and TCP traffic.
+
+-  Network resiliency features: setup retries, failovers, circuit breakers, and
+   fault injection.
+
+-  Security and authentication features: enforce security policies and enforce
+   access control and rate limiting defined through the configuration API.
+
+### Platform-agnostic service discovery {#discovery}
+
+Service discovery works in a similar way regardless of what platform you're
+using:
+
+1. The platform starts a new instance of a service which notifies its platform
+   adapter.
+
+1. The platform adapter registers the instance with the Pilot abstract model.
+
+1. **Pilot** distributes traffic rules and configurations to the Envoy proxies
+   to account for the change.
+
+The diagram shows how the platform adapters and Envoy proxies interact.
+
+{{< image width="40%"
+    link="./discovery.svg"
+    caption="Service discovery"
+    >}}
+
+Because the service discovery feature is platform-independent:
+
+-  A service mesh can include services across multiple platforms.
+
+-  Envoy proxies enforce the traffic rules, configurations, and load balancing
+   for all instances.
+
+You can use the Istio service discovery features with the features provided by
+platforms like Kubernetes for container-based applications. See your platform's
+documentation for more information.
+
+### Load balancing
+
+Using Istio, all traffic bound to a service goes through the appropriate Envoy
+proxy. Envoy proxies distribute the traffic across the instances in the calling
+service's load balancing pool, and update load balancing pools according to
+changes to the Pilot abstract model.
+
+Istio supports the following load balancing methods:
+
+-  Round robin: Requests are forwarded to instances in the pool in turn, and
+   the algorithm instructs the load balancer to go back to the top of the pool
+   and repeat.
+
+-  Random: Requests are forwarded at random to instances in the pool.
+
+-  Weighted: Requests are forwarded to instances in the pool according to a
+   specific percentage.
+
+-  Least requests: Requests are forwarded to instances with the least number of
+   requests. See the [Envoy load balancing documentation](https://www.envoyproxy.io/docs/envoy/v1.5.0/intro/arch_overview/load_balancing)
+   for more information.
+
+You can also choose to prioritize your load balancing pools based on geographic
+location. Visit the [operations guide](/docs/ops/traffic-management/locality-load-balancing/)
+for more information on the locality load balancing feature.
+
+### Example traffic configuration
+
+The following diagram shows a basic example of traffic management using Pilot
+and Envoy proxies:
+
+{{< image width="60%"
+    link="./routing-overview.svg"
+    caption="Traffic management example"
+    >}}
+
+To learn more about the traffic management resources shown, please continue to
+our [Traffic routing and configuration concept](/docs/concepts/traffic-management/routing/)
```

Comment:
I have a hard time telling in the github ui if there is extra whitespace on these links.  A highlight operation shows them with extra whitespace.